Web/Admin.UI/src/app/components/reports/generate-report/generate-report-form.component.ts

```
@@ -0,0 +1,209 @@
+    onEventGenerated(facilityId: string) {
+        this.facilityId = facilityId;
+    }
+
+    compareReportTypes(object1: any, object2: any) {
+        return (object1 && object2) && object1 === object2;
+    }
+
+    // Add patient name to array
+    addPatient(): void {
+        const patientNameControl = this.generateReportForm.get('patients');
+        if (patientNameControl && patientNameControl.valid) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)